dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -469,3 +470,45 @@ def test_security_temporary_defaults(EnvSpecificCluster, loop):
             future = client.submit(lambda x: x + 1, 10)
             result = future.result(timeout=30)
             assert result == 11
+
+
+def test_jobqueuecluster_from_name_for_existing_scheduling_systems():
+    for scheduling_system in CLUSTER_CLASSES.keys():
+        temporary_config = {
+            f"jobqueue.{scheduling_system}.cores": 8,
+            f"jobqueue.{scheduling_system}.memory": "24GB",
+            f"jobqueue.{scheduling_system}.disk": "1GB",
+        }
+        with dask.config.set(temporary_config):
+            cluster = JobQueueCluster.from_name(scheduling_system)
+            assert isinstance(cluster, CLUSTER_CLASSES[scheduling_system])
+
+
+def test_jobqueuecluster_from_name_for_custom_cluster():
+    temporary_config = {
```

Comment:
Here also you don't seem to check the custom config. Wasn't that mean to check that htcondor default config wasn't taken into account?